rocAL/rocAL/include/augmentations/effects_augmentations/node_fog.h

```diff
@@ -24,19 +24,18 @@ THE SOFTWARE.
 #include "node.h"
 #include "parameter_factory.h"
 #include "parameter_vx.h"
-class FogNode : public Node
-{
-public:
+class FogNode : public Node {
```

Comment:
@fiona-gladwin Add a line space between 26 & 27. The other files seem to have that

Reply:
Done